testdata/p4tc_samples_outputs/default_action_example.c

```diff
@@ -0,0 +1,532 @@
+#include "ebpf_kernel.h"
```

Comment:
are we sure we want to save the C files as well?
They can be tested in a different way, e.g., by compiling them.
Comparing them is not as useful, especially if they could change a lot when the compiler changes.